<!DOCTYPE html>
<html>
<head><title>R: Population and Evolutionary Genetics Analysis System</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body><div class="container">
<h1> Population and Evolutionary Genetics Analysis System
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;pegas&rsquo; version 1.3</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../doc/index.html">User guides, package vignettes and other documentation.</a></li>
<li><a href="../NEWS">Package NEWS</a>.</li>
</ul>

<h2>Help Pages</h2>


<p style="text-align: center;">
<a href="# "> </a>
<a href="#A">A</a>
<a href="#B">B</a>
<a href="#C">C</a>
<a href="#D">D</a>
<a href="#E">E</a>
<a href="#F">F</a>
<a href="#G">G</a>
<a href="#H">H</a>
<a href="#I">I</a>
<a href="#J">J</a>
<a href="#L">L</a>
<a href="#M">M</a>
<a href="#N">N</a>
<a href="#P">P</a>
<a href="#R">R</a>
<a href="#S">S</a>
<a href="#T">T</a>
<a href="#U">U</a>
<a href="#V">V</a>
<a href="#W">W</a>
<a href="#misc">misc</a>
</p>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="pegas-package.html">pegas-package</a></td>
<td>Population and Evolutionary Genetics Analysis System</td></tr>
</table>

<h2><a id="A">-- A --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="all.equal.haploNet.html">all.equal.haploNet</a></td>
<td>Compare Two Haplotype Networks</td></tr>
<tr><td style="width: 25%;"><a href="alleles2loci.html">alleles2loci</a></td>
<td>Build Loci Object From Matrix of Alleles</td></tr>
<tr><td style="width: 25%;"><a href="allelicrichness.html">allelicrichness</a></td>
<td>Allelic Richness and Rarefaction Plots</td></tr>
<tr><td style="width: 25%;"><a href="amova.html">amova</a></td>
<td>Analysis of Molecular Variance</td></tr>
<tr><td style="width: 25%;"><a href="haploNet.html">as.evonet.haploNet</a></td>
<td>Haplotype Networks</td></tr>
<tr><td style="width: 25%;"><a href="haploNet.html">as.igraph.haploNet</a></td>
<td>Haplotype Networks</td></tr>
<tr><td style="width: 25%;"><a href="conversion.html">as.loci</a></td>
<td>Conversion Among Allelic Data Classes</td></tr>
<tr><td style="width: 25%;"><a href="conversion.html">as.loci.character</a></td>
<td>Conversion Among Allelic Data Classes</td></tr>
<tr><td style="width: 25%;"><a href="conversion.html">as.loci.data.frame</a></td>
<td>Conversion Among Allelic Data Classes</td></tr>
<tr><td style="width: 25%;"><a href="conversion.html">as.loci.factor</a></td>
<td>Conversion Among Allelic Data Classes</td></tr>
<tr><td style="width: 25%;"><a href="conversion.html">as.loci.genind</a></td>
<td>Conversion Among Allelic Data Classes</td></tr>
<tr><td style="width: 25%;"><a href="conversion.html">as.loci.matrix</a></td>
<td>Conversion Among Allelic Data Classes</td></tr>
<tr><td style="width: 25%;"><a href="haploNet.html">as.network.haploNet</a></td>
<td>Haplotype Networks</td></tr>
<tr><td style="width: 25%;"><a href="haploNet.html">as.phylo.haploNet</a></td>
<td>Haplotype Networks</td></tr>
</table>

<h2><a id="B">-- B --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="bind.loci.html">bind.loci</a></td>
<td>Bind Loci Objects</td></tr>
<tr><td style="width: 25%;"><a href="by.loci.html">by.loci</a></td>
<td>Summary by Population or Other Factor</td></tr>
</table>

<h2><a id="C">-- C --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="bind.loci.html">cbind.loci</a></td>
<td>Bind Loci Objects</td></tr>
<tr><td style="width: 25%;"><a href="cophenetic.haploNet.html">cophenetic.haploNet</a></td>
<td>Cophenetic Matrix on Haplotype Networks</td></tr>
</table>

<h2><a id="D">-- D --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="diffHaplo.html">diffHaplo</a></td>
<td>Comparison Between Two Haplotypes</td></tr>
<tr><td style="width: 25%;"><a href="dist.asd.html">dist.asd</a></td>
<td>Allelic Sharing Distance</td></tr>
<tr><td style="width: 25%;"><a href="dist.hamming.html">dist.hamming</a></td>
<td>Hamming Distance</td></tr>
<tr><td style="width: 25%;"><a href="haplotype.loci.html">dist.haplotype.loci</a></td>
<td>Haplotype Extraction and Frequencies From Allelic Data</td></tr>
</table>

<h2><a id="E">-- E --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="edit.loci.html">edit.loci</a></td>
<td>Edit Allelic Data with R's Data Editor</td></tr>
<tr><td style="width: 25%;"><a href="utilities.html">expand.genotype</a></td>
<td>Utily Functions for pegas</td></tr>
</table>

<h2><a id="F">-- F --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="F234.html">F2</a></td>
<td>F-Statistics From Patterson et al</td></tr>
<tr><td style="width: 25%;"><a href="F234.html">F3</a></td>
<td>F-Statistics From Patterson et al</td></tr>
<tr><td style="width: 25%;"><a href="F234.html">F4</a></td>
<td>F-Statistics From Patterson et al</td></tr>
<tr><td style="width: 25%;"><a href="Fst.html">Fst</a></td>
<td>F-Statistics</td></tr>
</table>

<h2><a id="G">-- G --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="conversion.html">genind2loci</a></td>
<td>Conversion Among Allelic Data Classes</td></tr>
<tr><td style="width: 25%;"><a href="geod.html">geod</a></td>
<td>Geodesic Distances</td></tr>
<tr><td style="width: 25%;"><a href="geoTrans.html">geoTrans</a></td>
<td>Manipulate Geographical Coordinates</td></tr>
<tr><td style="width: 25%;"><a href="geoTrans.html">geoTrans2</a></td>
<td>Manipulate Geographical Coordinates</td></tr>
<tr><td style="width: 25%;"><a href="utilities.html">getAlleles</a></td>
<td>Utily Functions for pegas</td></tr>
<tr><td style="width: 25%;"><a href="utilities.html">getGenotypes</a></td>
<td>Utily Functions for pegas</td></tr>
<tr><td style="width: 25%;"><a href="options_themes_haploNet.html">getHaploNetOptions</a></td>
<td>Options to Plot haploNet Objects</td></tr>
<tr><td style="width: 25%;"><a href="VCFloci.html">getINFO</a></td>
<td>Information From VCF Files</td></tr>
<tr><td style="width: 25%;"><a href="amova.html">getPhi</a></td>
<td>Analysis of Molecular Variance</td></tr>
<tr><td style="width: 25%;"><a href="utilities.html">getPloidy</a></td>
<td>Utily Functions for pegas</td></tr>
</table>

<h2><a id="H">-- H --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="heterozygosity.html">H</a></td>
<td>Heterozygosity at a Locus Using Gene Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="heterozygosity.html">H.default</a></td>
<td>Heterozygosity at a Locus Using Gene Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="heterozygosity.html">H.loci</a></td>
<td>Heterozygosity at a Locus Using Gene Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="hap.div.html">hap.div</a></td>
<td>Haplotype Diversity</td></tr>
<tr><td style="width: 25%;"><a href="hap.div.html">hap.div.DNAbin</a></td>
<td>Haplotype Diversity</td></tr>
<tr><td style="width: 25%;"><a href="hap.div.html">hap.div.haplotype</a></td>
<td>Haplotype Diversity</td></tr>
<tr><td style="width: 25%;"><a href="haploFreq.html">haploFreq</a></td>
<td>Haplotype Frequencies With a Covariate</td></tr>
<tr><td style="width: 25%;"><a href="haploNet.html">haploNet</a></td>
<td>Haplotype Networks</td></tr>
<tr><td style="width: 25%;"><a href="haplotype.html">haplotype</a></td>
<td>Haplotype Extraction and Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="haplotype.html">haplotype.character</a></td>
<td>Haplotype Extraction and Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="haplotype.html">haplotype.DNAbin</a></td>
<td>Haplotype Extraction and Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="haplotype.loci.html">haplotype.loci</a></td>
<td>Haplotype Extraction and Frequencies From Allelic Data</td></tr>
<tr><td style="width: 25%;"><a href="haplotype.html">haplotype.numeric</a></td>
<td>Haplotype Extraction and Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="heterozygosity.html">heterozygosity</a></td>
<td>Heterozygosity at a Locus Using Gene Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="hw.test.html">hw.test</a></td>
<td>Test of Hardy-Weinberg Equilibrium</td></tr>
<tr><td style="width: 25%;"><a href="hw.test.html">hw.test.genind</a></td>
<td>Test of Hardy-Weinberg Equilibrium</td></tr>
<tr><td style="width: 25%;"><a href="hw.test.html">hw.test.loci</a></td>
<td>Test of Hardy-Weinberg Equilibrium</td></tr>
</table>

<h2><a id="I">-- I --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="utilities.html">is.phased</a></td>
<td>Utily Functions for pegas</td></tr>
<tr><td style="width: 25%;"><a href="utilities.html">is.snp</a></td>
<td>Utily Functions for pegas</td></tr>
<tr><td style="width: 25%;"><a href="utilities.html">is.snp.loci</a></td>
<td>Utily Functions for pegas</td></tr>
<tr><td style="width: 25%;"><a href="VCFloci.html">is.snp.VCFinfo</a></td>
<td>Information From VCF Files</td></tr>
</table>

<h2><a id="J">-- J --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="jaguar.html">jaguar</a></td>
<td>Jaguar Micro-Satellites</td></tr>
</table>

<h2><a id="L">-- L --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="LD.html">LD</a></td>
<td>Linkage Disequilibrium</td></tr>
<tr><td style="width: 25%;"><a href="LD.html">LD2</a></td>
<td>Linkage Disequilibrium</td></tr>
<tr><td style="width: 25%;"><a href="LDscan.html">LDmap</a></td>
<td>Multi-Locus Linkage Disequilibrium</td></tr>
<tr><td style="width: 25%;"><a href="LDscan.html">LDscan</a></td>
<td>Multi-Locus Linkage Disequilibrium</td></tr>
<tr><td style="width: 25%;"><a href="LDscan.html">LDscan.DNAbin</a></td>
<td>Multi-Locus Linkage Disequilibrium</td></tr>
<tr><td style="width: 25%;"><a href="LDscan.html">LDscan.loci</a></td>
<td>Multi-Locus Linkage Disequilibrium</td></tr>
<tr><td style="width: 25%;"><a href="stairway.html">lines.stairway</a></td>
<td>The Stairway Plot</td></tr>
<tr><td style="width: 25%;"><a href="read.loci.html">loci</a></td>
<td>Read Allelic Data Files</td></tr>
<tr><td style="width: 25%;"><a href="alleles2loci.html">loci2alleles</a></td>
<td>Build Loci Object From Matrix of Alleles</td></tr>
<tr><td style="width: 25%;"><a href="conversion.html">loci2genind</a></td>
<td>Conversion Among Allelic Data Classes</td></tr>
<tr><td style="width: 25%;"><a href="conversion.html">loci2SnpMatrix</a></td>
<td>Conversion Among Allelic Data Classes</td></tr>
</table>

<h2><a id="M">-- M --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="mjn.html">mjn</a></td>
<td>Median-Joining Network</td></tr>
<tr><td style="width: 25%;"><a href="MMD.html">MMD</a></td>
<td>Mismatch Distribution</td></tr>
<tr><td style="width: 25%;"><a href="mst.html">msn</a></td>
<td>Minimum Spanning Tree and Network</td></tr>
<tr><td style="width: 25%;"><a href="mst.html">mst</a></td>
<td>Minimum Spanning Tree and Network</td></tr>
<tr><td style="width: 25%;"><a href="mutations.html">mutations</a></td>
<td>Plot Mutations on Networks</td></tr>
</table>

<h2><a id="N">-- N --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="na.omit.loci.html">na.omit.loci</a></td>
<td>Missing Allelic Data</td></tr>
<tr><td style="width: 25%;"><a href="nuc.div.html">nuc.div</a></td>
<td>Nucleotide Diversity</td></tr>
<tr><td style="width: 25%;"><a href="nuc.div.html">nuc.div.DNAbin</a></td>
<td>Nucleotide Diversity</td></tr>
<tr><td style="width: 25%;"><a href="nuc.div.html">nuc.div.haplotype</a></td>
<td>Nucleotide Diversity</td></tr>
<tr><td style="width: 25%;"><a href="na.omit.loci.html">nullAlleles2NA</a></td>
<td>Missing Allelic Data</td></tr>
</table>

<h2><a id="P">-- P --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="pegas-package.html">pegas</a></td>
<td>Population and Evolutionary Genetics Analysis System</td></tr>
<tr><td style="width: 25%;"><a href="haploNet.html">plot.haploNet</a></td>
<td>Haplotype Networks</td></tr>
<tr><td style="width: 25%;"><a href="haplotype.html">plot.haplotype</a></td>
<td>Haplotype Extraction and Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="haplotype.loci.html">plot.haplotype.loci</a></td>
<td>Haplotype Extraction and Frequencies From Allelic Data</td></tr>
<tr><td style="width: 25%;"><a href="mjn.html">plot.mjn</a></td>
<td>Median-Joining Network</td></tr>
<tr><td style="width: 25%;"><a href="site.spectrum.html">plot.spectrum</a></td>
<td>Site Frequency Spectrum</td></tr>
<tr><td style="width: 25%;"><a href="stairway.html">plot.stairway</a></td>
<td>The Stairway Plot</td></tr>
<tr><td style="width: 25%;"><a href="summary.loci.html">plot.summary.loci</a></td>
<td>Print and Summaries of Loci Objects</td></tr>
<tr><td style="width: 25%;"><a href="sw.html">plot.sw</a></td>
<td>Sliding Windows</td></tr>
<tr><td style="width: 25%;"><a href="plotNetMDS.html">plotNetMDS</a></td>
<td>Plot Networks With MDS Layout</td></tr>
<tr><td style="width: 25%;"><a href="amova.html">print.amova</a></td>
<td>Analysis of Molecular Variance</td></tr>
<tr><td style="width: 25%;"><a href="haploNet.html">print.haploNet</a></td>
<td>Haplotype Networks</td></tr>
<tr><td style="width: 25%;"><a href="haplotype.html">print.haplotype</a></td>
<td>Haplotype Extraction and Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="summary.loci.html">print.loci</a></td>
<td>Print and Summaries of Loci Objects</td></tr>
<tr><td style="width: 25%;"><a href="summary.loci.html">print.summary.loci</a></td>
<td>Print and Summaries of Loci Objects</td></tr>
<tr><td style="width: 25%;"><a href="VCFloci.html">print.VCFinfo</a></td>
<td>Information From VCF Files</td></tr>
<tr><td style="width: 25%;"><a href="utilities.html">proba.genotype</a></td>
<td>Utily Functions for pegas</td></tr>
</table>

<h2><a id="R">-- R --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="R2.test.html">R2.test</a></td>
<td>Ramos-Onsins-Rozas Test of Neutrality</td></tr>
<tr><td style="width: 25%;"><a href="VCFloci.html">rangePOS</a></td>
<td>Information From VCF Files</td></tr>
<tr><td style="width: 25%;"><a href="allelicrichness.html">rarefactionplot</a></td>
<td>Allelic Richness and Rarefaction Plots</td></tr>
<tr><td style="width: 25%;"><a href="bind.loci.html">rbind.loci</a></td>
<td>Bind Loci Objects</td></tr>
<tr><td style="width: 25%;"><a href="read.gtx.html">read.gtx</a></td>
<td>Read Genetix Data Files</td></tr>
<tr><td style="width: 25%;"><a href="read.loci.html">read.loci</a></td>
<td>Read Allelic Data Files</td></tr>
<tr><td style="width: 25%;"><a href="read.vcf.html">read.vcf</a></td>
<td>Read Variant Calling Format Files</td></tr>
<tr><td style="width: 25%;"><a href="replot.html">replot</a></td>
<td>Edit the Layout of a Haplotype Network</td></tr>
<tr><td style="width: 25%;"><a href="allelicrichness.html">rhost</a></td>
<td>Allelic Richness and Rarefaction Plots</td></tr>
<tr><td style="width: 25%;"><a href="mst.html">rmst</a></td>
<td>Minimum Spanning Tree and Network</td></tr>
<tr><td style="width: 25%;"><a href="rr.test.html">rr.test</a></td>
<td>Tajima Relative Rate Test of Molecular Clock</td></tr>
<tr><td style="width: 25%;"><a href="Fst.html">Rst</a></td>
<td>F-Statistics</td></tr>
</table>

<h2><a id="S">-- S --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="VCFloci.html">selectQUAL</a></td>
<td>Information From VCF Files</td></tr>
<tr><td style="width: 25%;"><a href="options_themes_haploNet.html">setHaploNetOptions</a></td>
<td>Options to Plot haploNet Objects</td></tr>
<tr><td style="width: 25%;"><a href="site.spectrum.html">site.spectrum</a></td>
<td>Site Frequency Spectrum</td></tr>
<tr><td style="width: 25%;"><a href="site.spectrum.html">site.spectrum.DNAbin</a></td>
<td>Site Frequency Spectrum</td></tr>
<tr><td style="width: 25%;"><a href="site.spectrum.html">site.spectrum.loci</a></td>
<td>Site Frequency Spectrum</td></tr>
<tr><td style="width: 25%;"><a href="haplotype.html">sort.haplotype</a></td>
<td>Haplotype Extraction and Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="stairway.html">stairway</a></td>
<td>The Stairway Plot</td></tr>
<tr><td style="width: 25%;"><a href="subset.haplotype.html">subset.haplotype</a></td>
<td>Subsetting and Filtering Haplotypes</td></tr>
<tr><td style="width: 25%;"><a href="haplotype.html">summary.haplotype</a></td>
<td>Haplotype Extraction and Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="summary.loci.html">summary.loci</a></td>
<td>Print and Summaries of Loci Objects</td></tr>
<tr><td style="width: 25%;"><a href="sw.html">sw</a></td>
<td>Sliding Windows</td></tr>
<tr><td style="width: 25%;"><a href="sw.html">sw.default</a></td>
<td>Sliding Windows</td></tr>
<tr><td style="width: 25%;"><a href="sw.html">sw.DNAbin</a></td>
<td>Sliding Windows</td></tr>
</table>

<h2><a id="T">-- T --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="tajima.test.html">tajima.test</a></td>
<td>Test of the Neutral Mutation Hypothesis</td></tr>
<tr><td style="width: 25%;"><a href="theta.h.html">theta.h</a></td>
<td>Population Parameter THETA using Homozygosity</td></tr>
<tr><td style="width: 25%;"><a href="theta.k.html">theta.k</a></td>
<td>Population Parameter THETA using Expected Number of Alleles</td></tr>
<tr><td style="width: 25%;"><a href="theta.msat.html">theta.msat</a></td>
<td>Population Parameter THETA From Micro-Satellites</td></tr>
<tr><td style="width: 25%;"><a href="theta.s.html">theta.s</a></td>
<td>Population Parameter THETA using Segregating Sites</td></tr>
<tr><td style="width: 25%;"><a href="theta.s.html">theta.s.default</a></td>
<td>Population Parameter THETA using Segregating Sites</td></tr>
<tr><td style="width: 25%;"><a href="theta.s.html">theta.s.DNAbin</a></td>
<td>Population Parameter THETA using Segregating Sites</td></tr>
<tr><td style="width: 25%;"><a href="theta.tree.html">theta.tree</a></td>
<td>Population Parameter THETA Using Genealogy</td></tr>
<tr><td style="width: 25%;"><a href="theta.tree.html">theta.tree.hetero</a></td>
<td>Population Parameter THETA Using Genealogy</td></tr>
</table>

<h2><a id="U">-- U --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="utilities.html">unphase</a></td>
<td>Utily Functions for pegas</td></tr>
</table>

<h2><a id="V">-- V --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="VCFloci.html">VCFheader</a></td>
<td>Information From VCF Files</td></tr>
<tr><td style="width: 25%;"><a href="VCFloci.html">VCFlabels</a></td>
<td>Information From VCF Files</td></tr>
<tr><td style="width: 25%;"><a href="VCFloci.html">VCFloci</a></td>
<td>Information From VCF Files</td></tr>
</table>

<h2><a id="W">-- W --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="write.loci.html">write.loci</a></td>
<td>Write Allelic Data Files</td></tr>
<tr><td style="width: 25%;"><a href="amova.html">write.pegas.amova</a></td>
<td>Analysis of Molecular Variance</td></tr>
<tr><td style="width: 25%;"><a href="read.vcf.html">write.vcf</a></td>
<td>Read Variant Calling Format Files</td></tr>
</table>

<h2><a id="misc">-- misc --</a></h2>

<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="haplotype.html">[.haplotype</a></td>
<td>Haplotype Extraction and Frequencies</td></tr>
<tr><td style="width: 25%;"><a href="summary.loci.html">[.loci</a></td>
<td>Print and Summaries of Loci Objects</td></tr>
</table>
</div></body></html>
