<!DOCTYPE html>
<html>
<head><title>R: SNPs-Based Whole Genome Association Studies</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body><div class="container">
<h1> SNPs-Based Whole Genome Association Studies
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;SNPassoc&rsquo; version 2.1-2</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../doc/index.html">User guides, package vignettes and other documentation.</a></li>
</ul>

<h2>Help Pages</h2>


<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="setupSNP.html">$&lt;-.setupSNP</a></td>
<td>Convert columns in a dataframe to class 'snp'</td></tr>
<tr><td style="width: 25%;"><a href="inheritance.html">additive</a></td>
<td>Collapsing (or recoding) genotypes into different categories (generally two) depending on a given genetic mode of inheritance</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">additive.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">additive.WGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">as.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="association.html">association</a></td>
<td>Association analysis between a single SNP and a given phenotype</td></tr>
<tr><td style="width: 25%;"><a href="asthma.html">asthma</a></td>
<td>SNP data on asthma case-control study</td></tr>
<tr><td style="width: 25%;"><a href="BonferroniSig.html">Bonferroni.sig</a></td>
<td>Bonferroni correction of p values</td></tr>
<tr><td style="width: 25%;"><a href="inheritance.html">codominant</a></td>
<td>Collapsing (or recoding) genotypes into different categories (generally two) depending on a given genetic mode of inheritance</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">codominant.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">codominant.WGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="inheritance.html">dominant</a></td>
<td>Collapsing (or recoding) genotypes into different categories (generally two) depending on a given genetic mode of inheritance</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">dominant.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">dominant.WGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="inheritance.html">geneticModel</a></td>
<td>Collapsing (or recoding) genotypes into different categories (generally two) depending on a given genetic mode of inheritance</td></tr>
<tr><td style="width: 25%;"><a href="GenomicControl.html">GenomicControl</a></td>
<td>Population substructure</td></tr>
<tr><td style="width: 25%;"><a href="getGeneSymbol.html">getGeneSymbol</a></td>
<td>Get gene symbol from a list of SNPs</td></tr>
<tr><td style="width: 25%;"><a href="getNiceTable.html">getNiceTable</a></td>
<td>Get Latex output</td></tr>
<tr><td style="width: 25%;"><a href="getSignificantSNPs.html">getSignificantSNPs</a></td>
<td>Extract significant SNPs from an object of class 'WGassociation'</td></tr>
<tr><td style="width: 25%;"><a href="haplointeraction.html">haplo.interaction</a></td>
<td>Haplotype interaction with a covariate</td></tr>
<tr><td style="width: 25%;"><a href="HapMap.html">HapMap</a></td>
<td>SNPs from HapMap project</td></tr>
<tr><td style="width: 25%;"><a href="HapMap.SNPs.pos.html">HapMap.SNPs.pos</a></td>
<td>SNPs from HapMap project</td></tr>
<tr><td style="width: 25%;"><a href="inheritance.html">inheritance</a></td>
<td>Collapsing (or recoding) genotypes into different categories (generally two) depending on a given genetic mode of inheritance</td></tr>
<tr><td style="width: 25%;"><a href="int.html">int</a></td>
<td>Identify interaction term</td></tr>
<tr><td style="width: 25%;"><a href="interactionPval.html">interactionPval</a></td>
<td>Two-dimensional SNP analysis for association studies</td></tr>
<tr><td style="width: 25%;"><a href="intervals.html">intervals</a></td>
<td>Print ORs and 95% confidence intervals for an object of class 'haplo.glm'</td></tr>
<tr><td style="width: 25%;"><a href="intervals.html">intervals.haplo.glm</a></td>
<td>Print ORs and 95% confidence intervals for an object of class 'haplo.glm'</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">is.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="setupSNP.html">labels.setupSNP</a></td>
<td>Convert columns in a dataframe to class 'snp'</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">labels.WGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="LD.html">LD</a></td>
<td>max-statistic for a 2x3 table</td></tr>
<tr><td style="width: 25%;"><a href="LD.html">LD.setupSNP</a></td>
<td>max-statistic for a 2x3 table</td></tr>
<tr><td style="width: 25%;"><a href="LD.html">LD.snp</a></td>
<td>max-statistic for a 2x3 table</td></tr>
<tr><td style="width: 25%;"><a href="LD.html">LDplot</a></td>
<td>max-statistic for a 2x3 table</td></tr>
<tr><td style="width: 25%;"><a href="LD.html">LDtable</a></td>
<td>max-statistic for a 2x3 table</td></tr>
<tr><td style="width: 25%;"><a href="makegeno.html">make.geno</a></td>
<td>Create a group of locus objects from some SNPs, assign to 'model.matrix' class.</td></tr>
<tr><td style="width: 25%;"><a href="maxstat.html">maxstat</a></td>
<td>max-statistic for a 2x3 table</td></tr>
<tr><td style="width: 25%;"><a href="maxstat.html">maxstat.default</a></td>
<td>max-statistic for a 2x3 table</td></tr>
<tr><td style="width: 25%;"><a href="maxstat.html">maxstat.matrix</a></td>
<td>max-statistic for a 2x3 table</td></tr>
<tr><td style="width: 25%;"><a href="maxstat.html">maxstat.setupSNP</a></td>
<td>max-statistic for a 2x3 table</td></tr>
<tr><td style="width: 25%;"><a href="maxstat.html">maxstat.table</a></td>
<td>max-statistic for a 2x3 table</td></tr>
<tr><td style="width: 25%;"><a href="odds.html">odds</a></td>
<td>Extract odds ratios, 95% CI and pvalues</td></tr>
<tr><td style="width: 25%;"><a href="inheritance.html">overdominant</a></td>
<td>Collapsing (or recoding) genotypes into different categories (generally two) depending on a given genetic mode of inheritance</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">overdominant.WGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="permTest.html">permTest</a></td>
<td>Permutation test analysis</td></tr>
<tr><td style="width: 25%;"><a href="permTest.html">plot.permTest</a></td>
<td>Permutation test analysis</td></tr>
<tr><td style="width: 25%;"><a href="setupSNP.html">plot.setupSNP</a></td>
<td>Convert columns in a dataframe to class 'snp'</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">plot.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="interactionPval.html">plot.SNPinteraction</a></td>
<td>Two-dimensional SNP analysis for association studies</td></tr>
<tr><td style="width: 25%;"><a href="plotWGassociation.html">plot.WGassociation</a></td>
<td>Function to plot -log p values from an object of class 'WGassociation'</td></tr>
<tr><td style="width: 25%;"><a href="plotMissing.html">plotMissing</a></td>
<td>Plot of missing genotypes</td></tr>
<tr><td style="width: 25%;"><a href="haplointeraction.html">print.haploOut</a></td>
<td>Haplotype interaction with a covariate</td></tr>
<tr><td style="width: 25%;"><a href="intervals.html">print.intervals</a></td>
<td>Print ORs and 95% confidence intervals for an object of class 'haplo.glm'</td></tr>
<tr><td style="width: 25%;"><a href="maxstat.html">print.maxstat</a></td>
<td>max-statistic for a 2x3 table</td></tr>
<tr><td style="width: 25%;"><a href="permTest.html">print.permTest</a></td>
<td>Permutation test analysis</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">print.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="interactionPval.html">print.SNPinteraction</a></td>
<td>Two-dimensional SNP analysis for association studies</td></tr>
<tr><td style="width: 25%;"><a href="association.html">print.snpOut</a></td>
<td>Association analysis between a single SNP and a given phenotype</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">print.summary.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="tableHWE.html">print.tableHWE</a></td>
<td>Test for Hardy-Weinberg Equilibrium</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">print.WGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">pvalues</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">pvalues.WGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="qqpval.html">qqpval</a></td>
<td>Functions for inspecting population substructure</td></tr>
<tr><td style="width: 25%;"><a href="inheritance.html">recessive</a></td>
<td>Collapsing (or recoding) genotypes into different categories (generally two) depending on a given genetic mode of inheritance</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">recessive.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">recessive.WGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="related.html">related</a></td>
<td>Get related samples</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">reorder.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="resHapMap.html">resHapMap</a></td>
<td>SNPs from HapMap project</td></tr>
<tr><td style="width: 25%;"><a href="scanWGassociation.html">scanWGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="setupSNP.html">setupSNP</a></td>
<td>Convert columns in a dataframe to class 'snp'</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="SNPs.html">SNPs</a></td>
<td>SNPs in a case-control study</td></tr>
<tr><td style="width: 25%;"><a href="SNPs.info.pos.html">SNPs.info.pos</a></td>
<td>SNPs in a case-control study</td></tr>
<tr><td style="width: 25%;"><a href="sortSNPs.html">sortSNPs</a></td>
<td>Sort a vector of SNPs by genomic position</td></tr>
<tr><td style="width: 25%;"><a href="intervals.html">summary.haplo.glm</a></td>
<td>Print ORs and 95% confidence intervals for an object of class 'haplo.glm'</td></tr>
<tr><td style="width: 25%;"><a href="setupSNP.html">summary.setupSNP</a></td>
<td>Convert columns in a dataframe to class 'snp'</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">summary.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">summary.WGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="Tablemeanse.html">Table.mean.se</a></td>
<td>Descriptive sample size, mean, and standard error</td></tr>
<tr><td style="width: 25%;"><a href="TableNPer.html">Table.N.Per</a></td>
<td>Descriptive sample size and percentage</td></tr>
<tr><td style="width: 25%;"><a href="tableHWE.html">tableHWE</a></td>
<td>Test for Hardy-Weinberg Equilibrium</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">WGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">WGstats</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="setupSNP.html">[.setupSNP</a></td>
<td>Convert columns in a dataframe to class 'snp'</td></tr>
<tr><td style="width: 25%;"><a href="snp.html">[.snp</a></td>
<td>SNP object</td></tr>
<tr><td style="width: 25%;"><a href="WGassociation.html">[.WGassociation</a></td>
<td>Whole genome association analysis</td></tr>
<tr><td style="width: 25%;"><a href="setupSNP.html">[&lt;-.setupSNP</a></td>
<td>Convert columns in a dataframe to class 'snp'</td></tr>
<tr><td style="width: 25%;"><a href="setupSNP.html">[[&lt;-.setupSNP</a></td>
<td>Convert columns in a dataframe to class 'snp'</td></tr>
</table>
</div></body></html>
